client/dashboard/sites/settings/index.tsx

```diff
@@ -40,7 +40,15 @@ export default function SiteSettings( { siteSlug }: { siteSlug: string } ) {
 	}
 
 	return (
-		<PageLayout size="small" header={ <PageHeader title={ __( 'Settings' ) } /> }>
+		<PageLayout
+			size="small"
+			header={
+				<PageHeader
+					title={ __( 'Settings' ) }
+					description={ __( 'Configure your site’s general, server, and security settings.' ) }
```

Comment:
Maybe it's just me, but I think this description is very redundant 😄 I think this page is special as in it's just an "index" page for the individual pages. Also, for Simple sites you can't actually configure your server settings (they're all upsells).

So I'd recommend to just not have description on this page 😬 or if that's inconsistent, maybe just `Configure your site settings` 🤔